### ISSUE TITLE
Proxy role - proxy setting with no authentication requires password

### ISSUE THREAD

**Issue by ClaireHayard** [HewlettPackard/oneview-ansible-collection#176]
## Scenario/Intent

Hello,

I am using the `oneview_appliance_proxy_configuration` plugin to set up proxy settings on OneView with **no authentication**, however the task fails because it wants me to set the password parameter.

### main.yml

```
- name: Setup HTTP/HTTPS Proxy configuration (without Authentication)
  hpe.oneview.oneview_appliance_proxy_configuration:
    state: present
    config: "{{ config }}"
    data:
    data:
      server: "{{ proxy_server }}"
      port: "{{ port }}"
      communicationProtocol: "{{ protocol }}"
  delegate_to: localhost
```

## Environment Details

- OneView Ansible Collection: v6.6.0
- OneView Appliance Version: 6.10
- Ansible version: 2.9.6
- Python version: 3.8.10

## Expected Result

We should not have to add the password parameter to the data block in the task when we want to set up the proxy settings with no authentication.

## Actual Result

```
An exception occurred during task execution. To see the full traceback, use -vvv. The error was: KeyError: 'password'
failed: [localhost -> localhost] (item={'proxy_server': '<proxy_server_address>', 'port': '8080', 'protocol': 'HTTP'})
PLAY RECAP ***********************************************************************************************************
localhost                  : ok=3    changed=0    unreachable=0    failed=1    skipped=1    rescued=0    ignored=0   
```

**Comment by ClaireHayard** [comment]
Hello, any news on this ?

**Comment by alisha-k-kalladassery** [comment]
Hi Claire,

We have fixed this issue. The changes are now added to master branch and the same will be available in v7.2 release.
